lib/datasource/git-tags/index.spec.ts

```diff
@@ -51,7 +53,7 @@ describe('datasource/git-tags/index', () => {
           return Promise.resolve(lsRemote1);
         },
       });
-      const digest = await getDigest(
+      const digest = await new GitTagsDatasource().getDigest(
```

Comment:
Do we really need a new instance on every test? or can we declare on module scope?